soc/arm/st_stm32/stm32l0/soc.c

```diff
@@ -52,6 +52,13 @@ static int stm32l0_init(const struct device *arg)
 	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
 	LL_PWR_SetRegulVoltageScaling(LL_PWR_REGU_VOLTAGE_SCALE1);
 
+	/* On STM32L0, there are some hardfault when enabling DBGMCU bit:
+	 * Sleep, Stop or Standby. See #37119
```

Comment:
Please use full length adresses (https://github.com/zephyrproject-rtos/zephyr/pull/37119).
(For those not familiar with github)

Reply:
Done